docs/sidebars.js

```diff
@@ -16,6 +16,16 @@ module.exports = {
     "token-swap",
     "token-lending",
     "associated-token-account",
+    {
+      type: "category",
+      label: "Account Compression",
+      collapsed: true,
+      items: [
+        "account-compression",
+        "account-compression/concepts",
+        "account-compression/usage",
+      ]
+    },
```

Comment:
nit: can this be moved to the end, after the confidential extension?

Reply:
Fixed!